src/provider/proxyprovider.cpp

```diff
@@ -70,3 +70,45 @@ TransactionStatus ProxyProvider::getTransactionStatus(std::string const &txHash)
 
     return TransactionStatus(txStatus);
 }
+
+std::string ProxyProvider::getESDTTokenBalance(Address const &address, std::string const &token) const
+{
+    wrapper::http::Client client(m_url);
+    wrapper::http::Result const result = client.get("/address/" + address.getBech32Address() + "/esdt/" + token);
```

Comment:
Is string interpolation possible (here, below etc.)?

https://stackoverflow.com/a/57128991/1475331

Reply:
Unfortunately, it is only possible from C++20 (I've also tried it myself) :(
We are using C++14...